### PR TITLE
adding beta warning

### PR DESCRIPTION
**Pull request by necoline**

### CODE REVIEW

**Comment by ladyleet** [comment]
@necoline WOOHOO your first PR for RxJS docs!!!!!!

File: src/app/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     </mat-icon>
   </button>
   <span class="title" fxFlex> RxJS Docs </span>
+  <span class="title" fxFlex> WARNING: This is BETA site </span>
```
Comment (ladyleet) [inline src/app/toolbar/toolbar.component.html]: @necoline can we have a - For the current RxJS docs, visit http://reactivex.io/rxjs under that?

**Comment by btroncone** [comment]
Thanks! 👍

**Comment by ashwin-sureshkumar** [comment]
@necoline - Thanks for the PR. Just a small change in your commit message since it doesn't follow the commit message guidelines. 

Contribution guidelines has the commit message guideline information : https://github.com/ReactiveX/rxjs-docs/blob/master/CONTRIBUTING.md#commit-message-guidelines

Let me know if you need any help.

**Comment by ashwin-sureshkumar** [comment]
@necoline - Just following up on this PR. Let me know if you need any help

**Comment by ladyleet** [comment]
@necoline woot! merged <3

**Pull request merged**